routes/surveyRoutes.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose');
+const requireLogin = require('../middlewares/requireLogin');
+const requireCredits = require('../middlewares/requireCredits');
+const Mailer = require('../services/Mailer');
+const surveyTemplate = require('../services/emailTemplates/surveyTemplate');
+
+const Survey = mongoose.model('surveys');
+
+module.exports = app => {
+  app.get('/api/surveys/thanks', (req, res) => {
+    res.send('Thanks for voting!');
+  });
+
+  app.post('/api/surveys', requireLogin, requireCredits, async (req, res) => {
```

Comment:
This method is pretty long maybe you could have a SurveyHelper or something.

Reply:
thank you for the feedback... I'll put a TODO for this. 😄